docs/data/catalog/platform/gc-design-system/airtable.md

```diff
@@ -0,0 +1,115 @@
+# Platform / GC Design System / Airtable
+
+Dataset providing GC Design System client and engagement data exported from Airtable.
+
+Each row represents a client or user of the Government of Canada Design System, including their contact information, engagement status, team associations, meeting history, and support interactions.
+
+This dataset is represented in [Superset](https://superset.cds-snc.ca/) as the Physical dataset: `platform_gc_design_system_airtable`
+
+`Keywords`: Platform, GC Design System, Airtable, Clients, Engagement, Support
+
+---
+
+[:information_source: View the data pipeline](../../../pipelines/platform/gc-design-system/airtable.md)
+
+## Provenance
+
+This dataset is exported daily from the GC Design System Airtable base using the Airtable API. The data contains client engagement information, support interactions, and relationship data for users and organizations working with the Government of Canada Design System. More documentation on the pipeline can be found [here](../../../pipelines/platform/gc-design-system/airtable.md).
+
+* `Updated`: Daily at 5:00 AM UTC (Production only)
+* `Steward`: Platform Core Services
+* `Contact`: Slack channel #platform-core-services
+* `Location`: `cds-data-lake-transformed-production/platform/gc-design-system/airtable/*.jsonl`
+
+## Fields
+
+All fields are sourced directly from the GC Design System Airtable base. The exact schema is automatically discovered and may evolve as the Airtable structure changes.
+
+[Query to return example data](examples/airtable.sql) has been provided to explore the current structure.
+
+Here's a descriptive list of the core fields that are typically present:
+
+### Core Airtable Fields
+
+| Field | Type | Description |
+|-------|------|-------------|
+| id | string | Unique record identifier assigned by Airtable (format: recXXXXXXXXXXXXXX) |
+| created_time | timestamp | When the record was created in Airtable, in ISO 8601 format (UTC) |
+
+### Actual Fields in the Dataset
+
+*Note: The Lambda function normalizes column names by replacing spaces with underscores, removing quotes and parentheses, and converting to lowercase. Many fields are arrays since Airtable often stores linked records and multi-select fields as arrays:*
+
+| Field | Type | Description |
+|-------|------|-------------|
+| name | string | Client's full name |
+| email | array<string> | Array of email addresses (Airtable linked records) |
```

Comment:
We should chat about this with the team since we've been avoiding capturing people's name and email in the datasets so far.  Same comment would go for the other fields that look like they are people's names:
- primary_contact_on_team
- main_contact_on_meetings
- main_contact_on_engagement

To keep the linkages if the names are acting as primary keys we could apply a hashing function to them, and then with the emails, just preserve the email domain.